src/app/components/InlineLink/index.tsx

```diff
@@ -12,7 +12,7 @@ interface Props extends HTMLAttributes<HTMLElement> {
   className?: string;
   fontVariant?: FontVariant;
   size?: GelFontSize;
-  text: string;
+  text?: string;
```

Comment:
Should we maybe revert this - we should never have a link without text, so it makes sense that typescript would prevent it.